### PR TITLE
restoring Identifiers after ImportDeclaration transpile

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
@grant here is the fix to #16

### CODE REVIEW

**Comment by PopGoesTheWza** [comment]
@grant 
lastest commit fixes #18 (and adds tests)

about #16 : the transformer `ignoreImport` is correctly marking import statements an *NotEmitted* before transpiling. Still the module specifier (i.e. 'TypeScriptModule') is flagged and every reference is replaced by an 'autoGenerate' identifier. The solution is to add an 'after' transformer `restoreIdentifier` which restores the original identifier.

about #18 : `exportEsModuleNodeFilter` was too broad and thus triggering more than just `exports.__esModule`... statements

**Comment by PopGoesTheWza** [comment]
Hold on. I have one more change to commit about #16

**Comment by PopGoesTheWza** [comment]
@grant that should be it

File: src/index.ts [hunk 1]
```diff
@@ -43,6 +43,26 @@ const ts2gas = (source: string, transpileOptions: ts.TranspileOptions = {}) => {
       return (file: ts.SourceFile) => file;
     };
 
+    const restoreImportIdBuilder = (kind: ts.SyntaxKind, nodeFilter: NodeFilter): TransformerFactory =>
```
Comment (grant) [inline src/index.ts]: Add a comment as to what this does. Like the other functions.
Be sure to include `@param ...`

Comment (PopGoesTheWza) [reply src/index.ts]: @grant Will do after commute. When at home. Current ts2gas npm package is buggy and thus breaks clasp. Plus you need to add a tsc build when publishing.

**Pull request merged**